ci/kokoro/windows/CMakeLists.txt

```diff
@@ -0,0 +1,100 @@
+# ~~~
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ~~~
+
+cmake_minimum_required(VERSION 3.5)
+project(google-cloud-cpp-spanner-super-build CXX C)
+
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+set(GOOGLE_CLOUD_CPP_SPANNER_ROOT "${PROJECT_SOURCE_DIR}/../../..")
+list(APPEND CMAKE_MODULE_PATH "${GOOGLE_CLOUD_CPP_SPANNER_ROOT}/cmake")
+
+include(external/external-project-helpers)
+google_cloud_cpp_set_prefix_vars()
+
+include(ExternalProject)
+set(
+    GOOGLE_CLOUD_CPP_URL
+    "https://github.com/googleapis/google-cloud-cpp/archive/84772e182e5a82c9f839a9073cef88b9962c48cf.tar.gz"
```

Comment:
This is a different version than we're using in our other cmake builds:
https://github.com/googleapis/google-cloud-cpp-spanner/blob/fd3730b06613aa2a4db43e5845d7334c9c3aca13/cmake/external/google-cloud-cpp.cmake#L29

I'm curious: Can we use that same `cmake/external/google-cloud-cpp.cmake` file from this windows build too?

Reply:
Yeah I tried that, and getting into trouble. Maybe we can consolidate them later?